cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +182,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content string
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
+	errors = []error{}
+	f, err := os.Open(fcc.Path)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%s does not exists", fcc.Path))
+		return nil, errors
+	}
+
+	lr := io.LimitReader(f, 1)
```

Comment:
This assumes the file we are reading is one byte only, right?
I would prefer just a simple `ioutil.ReadFile` here that read the whole file and compared it to the `Content` byte array.